genesyscloud/external_contacts/resource_genesyscloud_externalcontacts_contact.go

```diff
@@ -117,6 +117,9 @@ func readExternalContact(ctx context.Context, d *schema.ResourceData, meta inter
 		resourcedata.SetNillableValueWithInterfaceArrayWithFunc(d, "facebook_id", externalContact.FacebookId, flattenSdkFacebookId)
 		resourcedata.SetNillableValue(d, "survey_opt_out", externalContact.SurveyOptOut)
 		resourcedata.SetNillableValue(d, "external_system_url", externalContact.ExternalSystemUrl)
+		if externalContact.ExternalOrganization != nil && externalContact.ExternalOrganization.Id != nil {
+			resourcedata.SetNillableValue(d, "external_organization_id", externalContact.ExternalOrganization.Id)
```

Comment:
You don't need to use `SetNillableValue` here. That function will do the same check you already did. You can use `d.Set` instead